docs/VTOL.md

```diff
@@ -113,10 +113,10 @@ set mc_iterm_relax = RPY
 save
 ```
 
-# STEP1: Configuring as a normal fixed-wing in Profile 1
+# STEP 1: Configuring as a normal fixed-wing in Profile 1
 
 1. **Select the fisrt Mixer Profile and PID Profile:**
-   - In the CLI, switch to the mixer_profile and pid_profile you wish to set first. You can also switch mixer_profile/pid_profile through gui if with aforementioned presets loaded.
+   - In the CLI, switch to the mixer_profile and pid_profile you wish to set first. You can also switch mixer_profile/pid_profile through gui with aforementioned presets loaded.
```

Comment:
Note, PID profile doesn't exist. It should be Control profile instead. I see this error exists all over this document.